analyzers/src/SonarAnalyzer.CFG/Extensions/ControlFlowGraphExtensions.cs

```diff
@@ -37,21 +38,21 @@ public static class ControlFlowGraphExtensions
                .Select(IFlowAnonymousFunctionOperationWrapper.FromOperation);
 
         // Similar to ControlFlowGraphExtensions.GetLocalFunctionControlFlowGraphInScope from Roslyn
-        public static ControlFlowGraph FindLocalFunctionCfgInScope(this ControlFlowGraph cfg, IMethodSymbol localFunction)
+        public static ControlFlowGraph FindLocalFunctionCfgInScope(this ControlFlowGraph cfg, IMethodSymbol localFunction, CancellationToken cancellationToken)
```

Comment:
Can we call it just `cancel` everywhere? 
```suggestion
        public static ControlFlowGraph FindLocalFunctionCfgInScope(this ControlFlowGraph cfg, IMethodSymbol localFunction, CancellationToken cancel)
```

Reply:
Will be done in a separate PR